js/analyses/getKeyphraseLength.js

```diff
@@ -0,0 +1,19 @@
+/** @module analyses/getKeyphraseLength */
```

Comment:
key phrase has a very specific meaning and it is not what we are working with. We're counting the number of words in a string or a text. So this function should be renamed to either `getWordCount` or `calculateWordCount`.